packages/integration-tests/tests/ui/withdraw.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, test } from "@playwright/test";
+
+import { config } from "./config";
+import { BlockExplorer, Buffer, Wallets } from "../../src/entities";
+import { Helper } from "../../src/helper";
+
+const bufferRoute = "src/playbook/";
+const helper = new Helper();
+let url: string;
+let bufferFile;
+let withdrawTxHash: string;
+let fromAddress: string;
+
+//@id1661
+test("Check Withdraw ETH transaction on BE", async ({ page }) => {
+  bufferFile = bufferRoute + Buffer.txEthWithdraw;
+  withdrawTxHash = await helper.getStringFromFile(bufferFile);
+  url = BlockExplorer.baseUrl + `/tx/${withdrawTxHash}` + BlockExplorer.localNetwork;
+  fromAddress = Wallets.richWalletAddress;
+  console.log(url);
+
+  await page.goto(url);
+
+  const hash = await page.locator(`//*[text()="Transaction Hash"]/..//..//*[text()="${withdrawTxHash}"]`).first();
+  const fromAddressElement = await page.locator(`text=${fromAddress}`).first();
+  const toAddressElement = await page
+    .locator(`//*[text()="To"]/..//..//..//*[text()="L1"]/..//*[text()="0x36615Cf349d...c049"]`)
```

Comment:
Perhaps we could be more flexible with such strict expressions. May be it is ok as it works now, but it will be hard to support in the future. I would like to discuss it with Anton.